server/routes/api/place.py

```diff
@@ -459,3 +462,61 @@ def api_ranking(dcid):
             result[label] = [x for x in result[label] if 'data' in x]
     result['label'] = [x for x in all_labels if x in result]
     return Response(json.dumps(result), 200, mimetype='application/json')
+
+
+@cache.memoize(timeout=3600 * 24) # Cache for one day.
+def get_state_code(dcid):
+    """Get state code for places that have a US State as one of its parent places
+
+    Args:
+        dcid: dcid of place we are looking for the state code for
+
+    Returns:
+        state code as a string or None if there is no US state code
+    """
+    state_code = None
+    for parent_place in json.loads(parent_places(dcid)):
```

Comment:
we should update parent_places and get_parent_place to support a list of dcids to reduce the number of mixer calls.